website/docs/release_notes.md

```diff
@@ -5,6 +5,97 @@ sidebar_position: 99
 
 # Release notes
 
+## [v0.1.11](https://github.com/VirtusLab/scala-cli/releases/tag/v0.1.11)
+
+### Add support for running multiple snippets of the same kind
+Scala CLI now allows passing and running multiple snippets. Each snippet is then treated as a
+separate input by Scala CLI.
+
+```bash
+scala-cli --scala-snippet '@main def main() = println(Messages.hello)' --scala-snippet 'object Messages { def hello = "Hello" }'
+```
+
+```text
+Hello
```

Comment:
when putting the output of a bash example in a doc, these chained docs should be wrapped in a `ChainedSnippets` component to render properly on our docs page.
You can refer [here](https://github.com/VirtusLab/scala-cli/blob/cad549fbc0de22e61588b8ec7d3394404724a3f8/website/docs/getting_started.md?plain=1#L16) for an example.

Now, the downside of using `ChainedSnippets` here is that it's our component and it will not work on the GitHub releases page when you copy the doc back there after the review is done as-is. 
So you might notice that in this particular doc we are fine with pasting examples such as this one in a plain text snippet, prefixing the bash prompt lines with a `$` ([here](https://github.com/VirtusLab/scala-cli/blob/cad549fbc0de22e61588b8ec7d3394404724a3f8/website/docs/release_notes.md?plain=1#L46) is an example for that).

Your call which way to fix that, really.

Reply:
Second approach implemented